### DISCUSSION TITLE
Cannot retrieve public data or do transactions after deploying to sepolia eth

### DISCUSSION THREAD

**Discussion by fox112358** [Cyfrin/foundry-full-course-cu#2743] [Q&A]
After deploying to sepolia and being able to see on sepolia eth scan that my smartcontract is there and the status is 'success' I am not able to do any interactions with contract itself from Remix.

call to public property is in pending state.

cannot do any transactions, evenmore it does not open up metamask to sign the transaction

![image](https://github.com/user-attachments/assets/327a27b3-9e93-4f1b-9306-50a31cf9719f)

tried clearing up the cookies and cache. Any ideas?

Thanks!

**Comment by EngrPips** [comment]
Hello @fox112358, It could be an internet connection problem. Do you think you have a stable internet connection?
